src/app/app.component.ts

```diff
@@ -7,4 +8,15 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'splitwise';
+  showLoginButton = true;
```

Comment:
Only variable declaration will be here. showLoginBUtton: boolean;

Reply:
Changes are done